client/src/reducers/roundApplication.ts

```diff
@@ -80,6 +81,20 @@ export const roundApplicationReducer = (
       };
     }
 
+    case ROUND_APPLICATION_ERROR_RESET: {
```

Comment:
I think we can skip this action because `ROUND_APPLICATION_LOADING ` already sets `error: undefined`, so the modal would close anyway. wdyt?

Reply:
this is what we use for closing the modal when done is clicked